components/ui/Newsletter.tsx

```diff
@@ -70,17 +76,24 @@ export default function Newsletter(
       return;
     }
 
-    modalAlertMensage.value = "Obrigado por se inscrever!";
-    modalMode.value = "success";
+    try {
+      loading.value = true;
+      await subscribe({ email, nome: name, celular: phone });
+      modalAlertMensage.value = "Obrigado por se inscrever!";
+      modalMode.value = "success";
```

Comment:
Quando voce tem um contexto desse prefira deixar as coisas juntas:
```suggestion
      modalAlert.value = { message: "Obrigado por se inscrever!", mode: "success" }
```

Reply:
feito